rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -330,6 +330,8 @@
         <rule id="hsearch-00110">
             <when>
                 <javaclass references="org.hibernate.search.backend.configuration.impl.IndexWriterSetting.{constant}">
+                    <location>ENUM_CONSTANT</location>
+                    <location>IMPORT</location>
```

Comment:
just limiting the search ;)

it should find all enumeration constants in Constructor calls, method invocations if the argument is the Enum type and in Variable/Field declarations.

I certainly can keep the rule without the definition of location, but this is just nice example how to use it and don't affect the result IMHO.